### PR TITLE
Add ibm-db package to Conda forge.

### PR DESCRIPTION
**Pull request by TeejIBM**
Signed-off-by: T.J. Griesenbrock <teej@ibm.com>

Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by TeejIBM** [comment]
Yeah, I definitely need some guidance on this.  This is just simply to add ibm-db to the Conda Forge package, so that a number of my co-workers can use this on their Anaconda installation.  I have no interaction with the development team for ibm-db itself.

**Comment by TeejIBM** [comment]
Corrected meta.yaml based on the results.  Still unsure what to do for test cases - none of the included test case are "simple checks" - it all requires a database.

**Comment by TeejIBM** [comment]
Okay, I added setuptools as a required package on the server, but on Win64, I still get the following failure:  

ModuleNotFoundError: No module named 'pkg_resources'

Searching for this indicates that the setuptools may have been incorrectly installed, and needs to be reinstalled.  Is there a known issue with the Win64 build server?  Please advise.

**Comment by TeejIBM** [comment]
Well, apparently either my tweak worked, OR there is a bad docker image floating around.  Please let me know if this suffices to add ibm-db package to Conda Forge.  Thanks.

File: recipes/ibm_db/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% set name = "ibm_db" %}
```
Comment (xhochy) [inline recipes/ibm_db/meta.yaml]: Looks like the official name on pypi is with a `-`. Should we also keep this?

**Comment by xhochy** [comment]
> Well, apparently either my tweak worked, OR there is a bad docker image floating around. Please let me know if this suffices to add ibm-db package to Conda Forge. Thanks.

No. Your package seems to require `setuptools` for determing its version thus you also need to list it as a dependency. `pkg_resources` is one of the modules that comes with `setuptools`.

File: recipes/ibm_db/meta.yaml (hunk 1 above)
Comment (TeejIBM) [reply recipes/ibm_db/meta.yaml]: That's how the actual file is named.  An earlier version of the meta.yaml had that name you suggested, but it didn't propagate correctly.  When importing packages, underscores are used.  Please advise.

Comment (xhochy) [reply recipes/ibm_db/meta.yaml]: As the documentation of `ibm_db` also is always using an `_` and the sdist is using, we can keep this here.

Note that the `conda` package name and the Python module name can be totally different.

**Pull request merged**